### PR TITLE
Scheduler classa

### PR DESCRIPTION
**Pull request by samuelszabo**
- reports.json config - list all reports (do we have an better way?)
- How to launch in Lambda? (eg. daily?)
- lambda_handler should raise an error (not 400 status code)

### CODE REVIEW

File: bin/lambda_handler.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@
 def lambda_handler(event:, context:)
   report_name = event['report']
 
-  return { statusCode: 400, body: "No report was specified" } if report_name.nil?
+  raise 'No report was specified' if report_name.nil?
```
Comment (samuelszabo) [inline bin/lambda_handler.rb:9]: @sukihub Totok. Lambda by mala crashnut, ak nema report_name.
Detto ak bude chyba v parseri/fetch.

Comment (sukihub) [reply bin/lambda_handler.rb:9]: @samuelszabo preco?

Comment (samuelszabo) [reply bin/lambda_handler.rb:9]: Lebo lambda má failnúť, ak má ostať entry v queue. Lambda ako taká, ak vráti 400, tak je to považované za OK stav (entry z queue sa zmaže).

Comment (samuelszabo) [reply bin/lambda_handler.rb:9]: nasta-backend - Lambda

O tomto v podstate hovorím. 404 ale Success.

**Pull request merged**